client/components/Nav.jsx

```diff
@@ -217,7 +217,7 @@ class Nav extends React.PureComponent {
               </li> }
               <li className="nav__dropdown-item">
                 <Link
-                  to="/p5/sketches"
+                  to={`/${process.env.EXAMPLE_USERNAME}/sketches`}
```

Comment:
l know this works fine in the development env:) but l don't know if it works in the production env